modules/examples/mlops-stepfunctions/app.py

```diff
@@ -0,0 +1,61 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+import aws_cdk
+import cdk_nag
+from aws_cdk import App
+from pydantic import ValidationError
+from sagemaker import image_uris  # type: ignore[import-untyped]
+from sagemaker.sklearn import defaults  # type: ignore[import-untyped]
+
+from settings import ApplicationSettings
+from stack import MLOPSSFNResources
+
+app = App()
+
+try:
+    app_settings = ApplicationSettings()
+except ValidationError as e:
```

Comment:
Nit - this try/except doesn't seem to be doing anything. Shall we remove it?

Reply:
Removed.